bin/ccs-push

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+USAGE_MSG="usage: $0 -h hostname -c camera -s subdir -f filename [-d]"
```

Comment:
I generally reserve uppercase for exported environment variables and use lowercase for internal local variables.  But we don't have standards for this (yet).

Reply:
Ha!  Figures...I had it as lower case when I first did it, and I switched it to upper.  I'll switch it back.